Makefile

```diff
@@ -17,7 +17,6 @@
 
 SHELL = /usr/bin/env bash
 .PHONY: all art cleanart version program lang path deps run race bindata generate build crossbuild clean test gofmt yamlfmt format docs rpmbuild mkdirs rpm srpm spec tar upload upload-sources upload-srpms upload-rpms copr
-.SILENT: clean bindata
```

Comment:
why are we dropping ths?

Reply:
> removed .SILENT from makefile as it being applied to only some targets makes it feel weird (I just learned about this option btw, feel free to comment on this change)

Adding `lang` to it instead of removing is also fine imho, but i was confused at first why `bindata` and `lang` had different behaviour while the recipies where the same.

Reply:
So what should we do here?
The reason (IIRC) these are here is because they're uninteresting, whereas getting output from the `lang` target is important.

Reply:
Lets keep it as is.